packages/repo-metadata-lint/src/validate.ts

```diff
@@ -102,6 +107,9 @@ export class Validate {
   async validApiShortNames() {
     const apiIndex = await this.getApiIndex();
     const apiShortNames = new Set<string>();
+    for (const shortname of EXTRA_ALLOWED_API_SHORTNAMES) {
```

Comment:
Pass EXTRA_ALLOWED_API_SHORTNAMES to the Set constructor, then no loop is necessary.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set

Reply:
Done.